### DISCUSSION TITLE
Get date and time in "normal" format

### DISCUSSION THREAD

**Discussion by mocart2** [andygrunwald/go-jira#610] [Q&A] [answered]
Hi!
I try to get issue created datetime
![image](https://user-images.githubusercontent.com/10042982/214958100-c22ac6f5-67ab-4fe2-9ad6-b56f873fca99.png)
```
fmt.Println(issue.Fields.Created)
```
but in output i see that:
```
{868000000 63810155809 0x104ddeca0}
```

how i can format to "normal" format, like `17.01.2023 17:15`

Thanks!

**Comment by vladimir-ch** [comment, marked as answer]
You're probably looking for something like
```
t := time.Time(issue.Fields.Created)
fmt.Println(t.Format("02.01.06 15:04"))
```
